### ISSUE TITLE
bug: Error Tesults results uploading

### ISSUE THREAD

**Issue by lucasdonato** [tesults/cypress-tesults-reporter#2]
Hi, when I have 1 spec with more than 2 its, it gives an error to upload test results

```
describe('visit google', () => {
    it(`google.com`, () => {
        cy.visit('https://google.com')
    })
    it(`google.com`, () => {
        cy.visit('https://google.com')
    })
})

```
![image](https://user-images.githubusercontent.com/12953400/221554117-87af3a64-78ce-45cb-92e8-9010d4332506.png)

**Comment by lucasdonato** [comment]
hi, I created a pull request with a possible fix, it worked perfectly for me.

https://github.com/tesults/cypress-tesults-reporter/pull/3/files

**Comment by ajeetd** [comment]
Thank you. Merged. Published update to npm, fix is in version 1.3.1.
